packages/account/src/providers/message.ts

```diff
@@ -71,3 +73,6 @@ export type MessageProof = {
 export type MessageStatus = {
   state: GqlMessageState;
 };
+
+export const isMessageCoin = (message: Message | MessageCoin): message is MessageCoin =>
```

Comment:
Nit but would prefer something like `isSpendableMessage` here due to the type `MessageCoin`.

Reply:
@danielbate, this was my initial idea, but it isn't entirely accurate semantically, as messages with data can also be spent.